src/main-process/parse-command-line.js

```diff
@@ -177,7 +193,25 @@ module.exports = function parseCommandLine (processArgs) {
   }
 }
 
-function normalizeDriveLetterName (filePath) {
+const readProjectSettingsSync = (filepath, executedFrom) => {
+  try {
+    const readPath = path.isAbsolute(filepath) ? filepath : path.join(executedFrom, filepath)
+    const contents = CSON.readFileSync(readPath)
+    if (contents.paths && contents.config) {
```

Comment:
@maxbrunsfeld what do you think about making `paths` optional and just making it default to the directory of the file?

Reply:
:+1: